src/HTTPCommands.hh

```diff
@@ -295,8 +295,13 @@ class HTTPUpload : public HTTPRequest {
 
 	virtual ~HTTPUpload();
 
-	virtual bool SendRequest(const std::string &payload, off_t offset,
-							 size_t size);
+	virtual bool SendRequest(const std::string &payload, bool final = true);
+
+	bool StartStreamingRequest(const std::string_view payload,
```

Comment:
Add doc string to the function.  I know we don't elsewhere but might as well start somewhere...